.github/workflows/pr-build-image.yaml

```diff
@@ -18,6 +18,7 @@ on:
   pull_request_target:
     paths-ignore:
       - 'docs/**'
+      - 'e2e-tests/docs/**'
```

Comment:
Maybe these should go under `docs/e2e-tests` so they're alongside all of the other repo documentation?

Reply:
agree and done!